changelogs/fragments/1795-list-elements-batch1.yml

```
@@ -0,0 +1,27 @@
+minor_changes:
+  - plugins/module_utils/oracle/oci_utils.py - elements of list parameter ``key_by`` is now validated (https://github.com/ansible-collections/community.general/pull/1795).
```

```suggestion
  - oci_utils module_utils - elements of list parameter ``key_by`` are now validated (https://github.com/ansible-collections/community.general/pull/1795).
```
(The `is` → `are` also for the other entries.)

---

instead of that, maybe:
>  `elements` of list param `xxx` is
?

---

or:
> the `elements` setting of the list params xxx is

---

I think it should still be `are` then, because elements is plural.

---

> > the `elements` setting of the list params xxx is

The `elements` setting is not validated, but the elements of the list option are validated.

---

fair enough, commit coming up